docs/Filters-and-Tags.md

```diff
@@ -16,13 +17,27 @@ If these filters do not meet your needs, you can also write your own filters.
 | get_parent_segment | Given a child segment name and overall message index, returns the first matched parent segment | `{% assign result = hl7v2Data \| get_parent_segment: 'childSegmentName', 3, 'parentSegmentName' -%}` |
 | has_segments | Checks if HL7 v2 message has segments | `{% assign result = hl7v2Data \| has_segments: 'segment1\|segment2\|...' -%}` | 
 
+### C-CDA specific filters
+| Filter | Description | Syntax |
+|-|-|-|
+| get_first_ccda_sections | Returns first instance (non-alphanumeric chars replace by '_' in name) of the sections | `{% assign firstSections = msg \| get_first_cda_sections: 'Problems' -%}` |
+| get_ccda_section_lists | Returns instance list (non-alphanumeric chars replace by '_' in name) for the given sections | `{% assign sections = msg \| get_cda_section_lists: 'Problems' -%}` |
+| get_first_ccda_sections_by_template_id | Returns first instance (non-alphanumeric chars replace by '_' in name) of the sections by template id | `{% assign firstSections = msg \| get_first_cda_sections_by_template_id: '2.16.840.1.113883.10.20.22.2.5.1' -%}` |
+
 ### String Filters
 | Filter | Description | Syntax |
 |-|-|-|
 | char_at | Returns char at position index | `{{ 'foo' \| char_at: 0 }} #=> 'f'` |
 | contains | Returns true if a string includes another string | `{{ 'foo' \| contains: 'fo' }} #=> true` |
 | escape_special_chars | Returns string with special chars escaped | `{{ '\E' \| escape_special_chars }} #=> '\\E'` |
 | unescape_special_chars | Returns string after removing escaping of special char | `{{ '\\E' \| unescape_special_chars }} #=> '\E'` |
+| match | Returns an array containing matches with a regular expression | `{% assign m = code \| match: '[0123456789.]+' -%}` |
+| to_json_string | Converts to JSON string | `{% assign msgJsonString = msg \| to_json_string -%}` |
+| base64_encode | Returns base64 encoded string | `{{ decodedData \| base64_encode }}` |
+| base64_decode | Returns base64 decoded string | `{{ encodedData \| base64_decode }}` |
+| sha1_hash | Returns SHA1 hash (in hex) of given string | `{{ inputData \| sha1_hash }}` |
+| gzip | Returns compressed string | `{{ uncompressedData \| gzip }}` |
+| gunzip_base64_string | Returns decompressed string | `{{ compressedData \| gunzip }}` |
```

Comment:
The example syntax in the third column is still `gunzip`. We can updated it to `gunzip_base64_string` as well.